issuing_using_did.md

```diff
@@ -160,63 +168,33 @@ The above structure will sufficiently allow for the identity be bounded for docu
 
 - Note that in this case, we have replaced the `DNS-TXT` record with a proof that the document store address can be associated with the DID
 
-### Revocation of signed document
+## Implementation Details
 
-Finally, to signal if the document may or may not be revoked, a compulsory key `revocation.type` must always be present. It may take on multiple types of values such as:
+TBD
 
-- `NONE`
-- `REVOCATION_STORE`
-- `SIMPLE_REVOCATION_LIST`
-- `REVOCATION_ENDPOINT`
-- etc
+## Non-breaking Upgrade
 
-A `revocation.type: "NONE"` must exist to ensure that the an end user did not obfuscate the revocation information to prevent his document from being revoked.
+TBD
 
-```json
-{
-  "schema": "tradetrust/v1.0",
-  "data": {
-    "issuers": [
-      {
-        "name": "TradeTrust Demo",
-        "revocation": {
-          "type": "REVOCATION_STORE",
-          "address": "0xabcd...1234"
-        }
-      }
-    ]
-  },
-  "privacy": {
-    "obfuscatedData": []
-  },
-  "signature": {
-    "type": "SHA3MerkleProof",
-    "targetHash": "61dc9186345e05cc2ae53dc72af880a3b66e2fa7983feaa6254d1518540de50a",
-    "proof": [],
-    "merkleRoot": "61dc9186345e05cc2ae53dc72af880a3b66e2fa7983feaa6254d1518540de50a"
-  },
-  "proof": {
-    "type": "DidGenericSignature",
-    "proofPurpose": "assertionMethod",
-    "signature": "<signed merkle root>",
-    "did": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6",
-    "didPublicKey": {
-      "id": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6#owner",
-      "type": "Secp256k1VerificationKey2018",
-      "ethereumAddress": "0xe6fe788d8ca214a080b0f6ac7f48480b2aefa9a6",
-      "owner": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6"
-    }
-  }
-}
-```
+### Difference with existing PR
 
-## Implementation Details
+[#91](https://github.com/Open-Attestation/oa-verify/pull/91)
+Instead of supporting a single signing method directly, the suggested approach is to support all the DID signing methods eventually, starting with did:ethr signing method. In addition, the specific public key used to sign the document is kept in the document to allow fo easy searching of matching key from the resolver.
```

Comment:
is the scope to only allow did:ethr for the moment ? the beginning of the document make it sounds like it's for any DID (I just want to make sure we are consistent accross the doc to avoid ambiguity)

Reply:
Yes it's for all, but because you have to implement the signature schemes separately we start with ethr